helix-term/src/commands/typed.rs

```diff
@@ -574,6 +574,46 @@ fn set_line_ending(
     Ok(())
 }
 
+fn set_borders(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+
+    if args.is_empty() {
+        let borders = (cx.editor.popup_border, cx.editor.menu_border);
+        cx.editor.set_status(match borders {
+            (true, true) => "all",
+            (true, false) => "popup",
+            (false, true) => "menu",
+            (false, false) => "none",
+        });
+
+        return Ok(());
+    }
+
+    let arg = args.get(0).context("argument missing")?.to_lowercase();
+
+    if let Some(borders) = match arg {
+        arg if arg.starts_with("all") => Some((true, true)),
+        arg if arg.starts_with("popup") => Some((true, false)),
+        arg if arg.starts_with("menu") => Some((false, true)),
+        arg if arg.starts_with("none") => Some((false, false)),
+        _ => None,
+    } {
+        cx.editor.popup_border = borders.0;
+        cx.editor.menu_border = borders.1;
+    } else {
+        cx.editor
+            .set_status("Valid options are: none, popup, menu, all");
+    };
+
+    Ok(())
+}
+
```

Comment:
Does this need its own command? Wouldn't just doing `:set popup-border <choice>` work?

Reply:
Yeah, i deleted this command now